feature/MTM-46344-count-of-device-in-group-incorrect-docs-update (#502)
SoftwareAG/c8y-docs: content/users-guide/cockpit-bundle/managing-assets.md

<img src="/images/users-guide/cockpit/cockpit-groups-subassets.png" name="Subassets"/>

{{< c8y-admon-info >}}

> The count displayed on top of the table on the **Subassets** tab shows the total number of child assets assigned to the current group. Any type of managed object can be a child asset. For more details on the counting of objects refer to [retrieve all child assets of a specific managed object](https://cumulocity.com/api/10.11.0/#operation/getManagedObjectChildAssetsResource) in the {{< openapi >}}.

Reviewer: ```suggestion
The count displayed on top of the table on the **Subassets** tab shows the total number of child assets assigned to the current group. Any type of managed object can be a child asset. For more details on the counting of objects refer to [retrieve all child assets of a specific managed object](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#operation/getManagedObjectChildAssetsResource) in the {{< openapi >}}.
```

Author: ok

> Subassets count displayed on top of the table of **Subassets** tab is showing total number of child assets assigned to current group. Any type of managed object can be a child asset. For more details on counting of objects please check OpenAPI specification for "[retrieve all child assets of a specific managed object](https://cumulocity.com/api/10.11.0/#operation/getManagedObjectChildAssetsResource)"

Reviewer 2: This is actually supposed to work by using the standard URL (without version). @rssag @manasesjesus Any update here? Who is taking care that this gets implemented?

> Subassets count displayed on top of the table of **Subassets** tab is showing total number of child assets assigned to current group. Any type of managed object can be a child asset. For more details on counting of objects please check OpenAPI specification for "[retrieve all child assets of a specific managed object](https://cumulocity.com/api/10.11.0/#operation/getManagedObjectChildAssetsResource)"

Reviewer: suggestion for the time being would be to use the standard URL variable, then the current version variable.

i notified jacek and martin about the need for another server URL.

> Subassets count displayed on top of the table of **Subassets** tab is showing total number of child assets assigned to current group. Any type of managed object can be a child asset. For more details on counting of objects please check OpenAPI specification for "[retrieve all child assets of a specific managed object](https://cumulocity.com/api/10.11.0/#operation/getManagedObjectChildAssetsResource)"

Reviewer 2: ```suggestion
The count displayed on top of the table on the **Subassets** tab shows the total number of child assets assigned to the current group. Any type of managed object can be a child asset. For more details on the counting of objects refer to [retrieve all child assets of a specific managed object](https://cumulocity.com/api/10.11.0/#operation/getManagedObjectChildAssetsResource) in the {{< openapi >}}.
```

> Subassets count displayed on top of the table of **Subassets** tab is showing total number of child assets assigned to current group. Any type of managed object can be a child asset. For more details on counting of objects please check OpenAPI specification for "[retrieve all child assets of a specific managed object](https://cumulocity.com/api/10.11.0/#operation/getManagedObjectChildAssetsResource)"

Author: where do you have description for standard variables which can be used in documentation? I didn't find anything in README.md

> Subassets count displayed on top of the table of **Subassets** tab is showing total number of child assets assigned to current group. Any type of managed object can be a child asset. For more details on counting of objects please check OpenAPI specification for "[retrieve all child assets of a specific managed object](https://cumulocity.com/api/10.11.0/#operation/getManagedObjectChildAssetsResource)"

Author: applied

> The count displayed on top of the table on the **Subassets** tab shows the total number of child assets assigned to the current group. Any type of managed object can be a child asset. For more details on the counting of objects refer to [retrieve all child assets of a specific managed object](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#operation/getManagedObjectChildAssetsResource) in the {{< openapi >}}.

Reviewer 3: ```suggestion
The count displayed on top of the table on the **Subassets** tab shows the total number of child assets assigned to the current group. Any type of managed object can be a child asset. For more details on the counting of objects refer to the operation [Retrieve all child assets of a specific managed object](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#operation/getManagedObjectChildAssetsResource) in the {{< openapi >}}.
```

Author: ok
